transformer/cluttered_mnist_rnn.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep 27 20:19:21 2016
+
+@author: serag
+"""
+
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import tensorflow as tf
+from spatial_transformer import transformer
+from tensorflow.python.ops import rnn,rnn_cell
```

Comment:
This doesn't work for tf 1.0 anymore, please also stay consistent with the empty spaces